pkg/backend/display/progress.go

```diff
@@ -315,7 +315,7 @@ func RenderProgressEvents(
 	o.RenderOnDirty = false
 	o.IsInteractive = true
 
-	printPermalinkInteractive(o.term, o, permalink)
+	printPermalinkInteractive(o.term, o, permalink, "")
```

Comment:
Looks like there's a new use of `printPermalinkInteractive` below that also now needs to be updated.

```
backend/display/progress.go:377:39: not enough arguments in call to printPermalinkInteractive
        have (terminal.Terminal, Options, string)
        want (terminal.Terminal, Options, string, string)
```